docs/reference/troubleshooting.md

```
@@ -29,16 +29,29 @@ dumber doctor --media   # GStreamer/VA-API only
 
 **Solution:**
 1. Run `dumber doctor --media`
-2. Check hardware decoding:
+2. Enable diagnostic mode for startup media warnings:
+   ```toml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)